extensions/amp-consent/0.1/consent-policy-manager.js

```diff
@@ -151,6 +157,26 @@ export class ConsentPolicyManager {
     });
   }
 
+  /**
+   * Wait for policy to resolve and check if it should be unblocked
+   * @param {string} policyId
+   * @return {!Promise<boolean>}
+   */
+  whenPolicyUnblock(policyId) {
+    let predefined;
+    if (policyId == UNBLOCK_ON_ALL_POLICY) {
+      predefined = policyId;
+      policyId = 'default';
+    }
+    return this.whenPolicyResolved(policyId).then(state => {
```

Comment:
ah, a bit weird. so the policyId has to be 'default'? If this only make sense when we later open up custom policy, we can simplify the code a bit to reduce confusion for now.

Reply:
Agree it's a bit weird. As discussed offline I made two functions `whenPolicyResolved` and `whenPolicyUnblock` independent of each other. Lot of duplicate code though, but since it's only experimental flag related I guess it's OK : )